### PR TITLE
update versions

### PR DESCRIPTION
**Pull request by johnynek**

### CODE REVIEW

File: language_generators/scala-defref-extractor/src/main/scala/io/bazeltools/buildgen/scaladefref/ScalaSourceEntityExtractor.scala [hunk 1]
```diff
@@ -118,7 +119,7 @@ object ScalaSourceEntityExtractor {
     ): Iterator[A] =
       for {
         importStmt <- imports.iterator
-        Importer(ref, importees) <- importStmt.importers.iterator
+        case Importer(ref, importees) <- importStmt.importers.iterator
```
Comment (johnynek) [inline language_generators/scala-defref-extractor/src/main/scala/io/bazeltools/buildgen/scaladefref/ScalaSourceEntityExtractor.scala:122]: scala 3 wants this for filtering.

File: language_generators/scala-defref-extractor/src/main/scala/io/bazeltools/buildgen/scaladefref/ScalaSourceEntityExtractor.scala [hunk 2]
```diff
@@ -15,8 +15,9 @@ import scala.meta.{
   Source,
   Term,
   Tree,
-  Pat
+  Pat,
 }
+import scala.meta.parsers.XtensionParseInputLike
```
Comment (johnynek) [inline language_generators/scala-defref-extractor/src/main/scala/io/bazeltools/buildgen/scaladefref/ScalaSourceEntityExtractor.scala:20]: scala 3 wants this import, it doesn't find the enrichment on its own.

File: language_generators/scala-defref-extractor/src/main/scala/io/bazeltools/buildgen/scaladefref/ScalaSourceEntityExtractor.scala [hunk 3]
```diff
@@ -602,8 +603,11 @@ object ScalaSourceEntityExtractor {
       case Type.With(left, right) =>
         log(s"Type.With($left, $right)")
         List(left, right).traverse_(inspect)
-      case Type.Placeholder(bounds) =>
```
Comment (johnynek) [inline language_generators/scala-defref-extractor/src/main/scala/io/bazeltools/buildgen/scaladefref/ScalaSourceEntityExtractor.scala:605]: Placeholder was deprecated.

**Review by ianoc — APPROVED** [review]

**Comment by ianoc** [comment]
Confirmed this runs against our internal repo

**Pull request merged**